### PR TITLE
Alex chen2

### PR DESCRIPTION
**Pull request by zalexchen**
Added Dynamically binding for RecyclerView to display listing of StudentOrganizations from Firebase.

Added onClick navigation for respective Student Org listed to navigate to their respective Student Org Profile. (Really just populating a template for Student Org Profiles with information for that particular student org reference).

Converted OrganizationProfile to extend Fragment instead of Activity

Noticed: MainActivity and OrganizationProfile currently unused (MainActivity was depreciated with the introduction of Fragments)

TODO: Add President, Vice President, and Phone to StudentOrganization data model.
Add organization description to info
Add navigation from info to posts and back.
Add posts

### CODE REVIEW

File: app/src/main/java/com/example/warriorsocial/ui/login/RegistrationChecker.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import android.content.Intent;
+import android.text.TextUtils;
+import android.widget.EditText;
+
+import com.example.warriorsocial.BottomActivity;
+import com.google.firebase.auth.FirebaseAuth;
+
+public class RegistrationChecker {
+    public RegistrationChecker() {}
+
```
Comment (ChloeMJM) [inline app/src/main/java/com/example/warriorsocial/ui/login/RegistrationChecker.java:13]: Does this need to be merged with dev? Or is this just for unit testing?

**Comment by rahulsheth1** [comment]
Everything looks really good, can see a lot of progress on files your working on.

**Comment by ChloeMJM** [comment]
I looked it over and don't see any issues! LGTM

**Comment by RichSosa28** [comment]
No issues founded when i  tested. Great job with the implementations!

**Pull request merged**